File: php/examples/post_companies.php
<?php
require_once 'base_request.php';

// Example of POST /companies

$debug = true;
$company_name 		= $_ENV['COMPANY_NAME'];
$home_state       = $_ENV['JURISDICTION'];
$entity_type 	    = $_ENV['ENTITY_TYPE'];

$request_data = array(
    'name' => $company_name,
    'home_state' => $home_state,
    'entity_type' => $entity_type,
);

$companies_data = array(
  'companies' => array($request_data)
);

$request_params = '';

$json_body = json_encode($companies_data);

send_request('POST', '/companies', $request_params, $json_body);

File: ruby/examples/post_companies.rb
require_relative '../request.rb'

# Example of POST /companies

NAME	= ENV['COMPANY_NAME']
ENTITY_TYPE = ENV['ENTITY_TYPE']
JURISDICTION 	= ENV['JURISDICTION']

params = { companies: [
	{ 
		name: NAME,
		entity_type: ENTITY_TYPE,
		home_state: JURISDICTION
	}
]}.to_json

BaseRequestRoute.request(:post, '/companies', body: params)

